pkg/tests/tasks/traffic/circuit_breaking_test.go

```diff
@@ -39,76 +46,95 @@ func cleanupCircuitBreaking() {
 }
 
 func TestCircuitBreaking(t *testing.T) {
-	test.NewTest(t).Id("T6").Groups(test.Full, test.InterOp).NotRefactoredYet()
+	NewTest(t).Id("T6").Groups(Full, InterOp).Run(func(t TestHelper) {
+		hack.DisableLogrusForThisTest(t)
+		ns := "bookinfo"
+		t.Cleanup(func() {
+			oc.RecreateNamespace(t, ns)
+		})
+
+		app.InstallAndWaitReady(t, app.Httpbin(ns), app.Fortio(ns))
+
+		t.NewSubTest("TrafficManagement_tripping").Run(func(t TestHelper) {
```

Comment:
This shouldn't be a subtest, since it's the only one.

Reply:
But we can add more subtest later to test others circuit break combinations. But ok, meanwhile I'm to delete the subtest

Reply:
Done